src/namer/namer.ts

```diff
@@ -1,13 +1,13 @@
-/*---------------------------------------------------------------------------------------------
+/*  ---------------------------------------------------------------------------------------------
```

Comment:
Are all these formatting changes from a tool?

Reply:
I was playing around with the linter. I'll take it away for now. ESLint rules said that there should be a space or tab after /*

Reply:
It's not all formatted completely according to the linter rules

Reply:
![image](https://user-images.githubusercontent.com/9859037/73579566-94e2c100-4437-11ea-8283-ced89df7dca3.png)

Actually I'm thinking of leaving it unless you have any objections

Reply:
I don't however if this is what the linter wants we should enable it and fix up all the source files to conform (can happen in a separate PR).

Reply:
Agreed, there are definitely a few more things that the linter has brought up. I'll add those changes to another PR.